server/api/github/callback.ts

```diff
@@ -14,11 +18,33 @@ export default defineEventHandler(async (event) => {
       code,
     },
   });
+  const response: any = await $fetch('https://github.com/login/oauth/access_token', {
+    method: 'POST',
+    body: {
+      client_id: config.public.github.clientId,
+      client_secret: config.github.clientSecret,
+      code,
+    },
+  });
```

Comment:
duplicated

Reply:
@anbraten yeah I saw. This new merge editor in vs code is very hard to use. I am fixing it.